src/tenant/models.py

```diff
@@ -44,15 +45,15 @@ class Tenant(TenantMixin):
     )
     owner_email = models.EmailField(null=True, blank=True)
     max_active_users = models.SmallIntegerField(
-        default=40,
+        default=5,
         help_text="The maximum number of users that can be active on this deck; -1 = unlimited."
     )
     max_quests = models.SmallIntegerField(
         default=100,
         help_text="The maximum number of quests that can be active on this deck (archived quests are considered inactive); -1 = unlimited."
     )
     trial_end_date = models.DateField(
-        blank=True, null=True,
+        default=(timezone.now() + timezone.timedelta(days=60)),
```

Comment:
This won't work because the function will be called when the server starts up, not when the tenant is created.
Fro example compare: [`default=timezone.now` vs `default = timzone.now()`](https://stackoverflow.com/questions/38237777/django-timezone-now-vs-timezone-now)  (I may have come across this before ;)